pkgs/servers/home-assistant/custom-components/gpio/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, home-assistant
+, buildHomeAssistantComponent
+, fetchFromGitea
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "raboof";
+  domain = "gpio";
+  version = "fe6adc4d7f7ac9a823ca7e2aecb7097d0417e5b7";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "raboof";
+    repo = "ha-gpio";
+    rev = version;
+    hash = "sha256-iLuTDqm/HuztMS//nSDrxxGAAs/rp0ODTtnTOQ5eAm4=";
+  };
+
+  propagatedBuildInputs = [ home-assistant.python.pkgs.libgpiod ];
```

Comment:
```suggestion
  propagatedBuildInputs = [ libgpiod ];
```

Reply:
ah, so confusing to have different functions called `callPackage` - but nice ;)